homeassistant/components/fronius/sensor.py

```diff
@@ -145,21 +147,32 @@ def data(self):
         """Return the state attributes."""
         return self._fetched
 
+    @property
+    def available(self):
```

Comment:
This class is the adapter, ie not the entity. The `available` property needs to go in the Entity class.

Reply:
Right, sorry I missed that :)